resin/models/environment_variables.py

```diff
@@ -27,6 +28,17 @@ def __init__(self):
         self.device = Device()
         self.settings = Settings()
 
+    def __fix_device_env_var_name_key(self, env_var):
```

Comment:
Double underscore named attributes are reserved by Python. Better use a single underscore to mark it as internal.